components/product/ProductCashback.tsx

```diff
@@ -0,0 +1,25 @@
+import { Product } from "apps/commerce/types.ts";
+import Icon from "site/components/ui/Icon.tsx";
+
+interface Props {
+  product: Product;
+  percentage: number;
+}
+
+export default function ProductCashback({ product, percentage }: Props) {
+  const showCashback = percentage > 0 &&
+    product.isVariantOf?.additionalProperty.find((property) =>
+      property.name?.toLowerCase() === "mostrar cashback" &&
+      property.value?.toLowerCase() === "sim"
+    );
```

Comment:
Caso a propriedade "mostrar cashback" seja encontrada com o valor `não`, a iteração vai seguir buscando pelo elemento, pois o find ainda não o encontrou. Sugiro encontrar o valor primeiro e, em seguida, validar seu valor.
```suggestion
    product.isVariantOf?.additionalProperty.find((property) =>
      property.name?.toLowerCase() === "mostrar cashback"
    )?.value.toLowerCase() === "sim";
```